docs/tables/turbot_policy_value.md

```diff
@@ -0,0 +1,93 @@
+# Table: turbot_resource
+
+Resources in Turbot represent cloud configuration items such as users,
+networks, servers, etc.
+
+It is recommended that queries to this table should include (usually in the `where` clause) at least one
+of these columns: `id`, `resource_type_id`, `resource_type_uri` or `filter`.
+
+A Policy Value is the effective policy setting on an instance of a resource type. Every resource that is targeted by a given policy setting will have its own value for that policy, which will be the resultant calculated policy for the "winning" policy in the hierarchy.
+
+Policy settings are inherited through the resource hierarchy, and values for a resource are calculated according to policy settings at or above it in the resource hierarchy. For example, a policy setting at the Turbot level will be inherited by all resources below.
+
+It is recommended that queries to this table should include (usually in the `where` clause) at least one
+of these columns: `state`,`policyTypeId`, `resource_type_id`, `resource_type_uri` or `filter`.
+
+## Examples
+
+### List policy values by policy type id
+
+```sql
+select
+  id,
+  state,
+  is_default,
+  is_calculated,
+  policy_type_id,
+  type_mod_uri
+from
+  turbot_policy_value
+where
+  policy_type_id = 123456789;
+```
+
+### List policy values by resource id
+
+```sql
+select
+  id,
+  state,
+  is_default,
+  is_calculated,
+  resource_id,
+  type_mod_uri
+from
+  turbot_policy_value
+where
+  resource_id = 123456789;
+```
+
+### List policy values by resource type id
+
+```sql
+select
+  id,
+  state,
+  is_default,
+  is_calculated,
+  resource_type_id,
+  type_mod_uri
+from
+  turbot_policy_value
+where
+  resource_type_id = 123456789;
+```
+
+### Filter for policy values using Turbot filter syntax
+
+```sql
+select
+  id,
+  state,
+  is_default,
+  is_calculated,
+  policy_type_id,
+  resource_id,
+  resource_type_id
+from
+  turbot_policy_value
+where
+  filter = 'policyTypeId:123456789';
+```
+
+### Extract all policy values from Turbot
```

Comment:
I don't believe we should include this example - policy values are set for every policy type on every applicable resource. I find it extremely rare that I'd want to see all policy values (policy settings or resources make more sense).

Reply:
Removed this example